test/support/flow_test_helper.rb

```diff
@@ -98,9 +98,10 @@ def testing_node
 
     def add_responses(responses)
       question_names = flow.questions.map(&:name)
+      valid_parameters = question_names + flow.additional_parameters
 
       responses.each do |key, value|
-        raise "Invalid question #{key}" unless question_names.include?(key.to_sym)
+        raise "Invalid response key #{key}" unless valid_parameters.include?(key.to_sym)
```

Comment:
I like the explicitness of the original error, so how about:
```suggestion
        raise "Invalid question or query parameter: #{key}" unless valid_parameters.include?(key.to_sym)
```